.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+<!-- Short description of the PR. What does it do? -->
+This PR implements a new data enhancement to...
+
+## Categories
+<!-- Select the categories that your PR better fits on -->
+
+- [ ] Bugfix
+- [X] Enhancement
+- [ ] Backporting
+- [ ] Others (CI, Infrastructure, Documentation)
+
+## Sample archive
+
+- `path/to/sample_data.json`
+
+## Documentation
+
+- `path/to/documentation.md`
+
+## Tests
```

Comment:
Too general. We should explicitly say unit tests because we don't usually add any e2e tests